DataFormats/Scouting/interface/Run3ScoutingPhoton.h

```diff
@@ -0,0 +1,96 @@
+#ifndef DataFormats_Run3ScoutingPhoton_h
+#define DataFormats_Run3ScoutingPhoton_h
+
+#include <vector>
+
+// Class for holding photon information, for use in data scouting
+// IMPORTANT: the content of this class should be changed only in backwards compatible ways!
+class Run3ScoutingPhoton {
+public:
+  //constructor with values for all data fields
+  Run3ScoutingPhoton(float pt,
+                     float eta,
+                     float phi,
+                     float m,
+                     float sigmaIetaIeta,
+                     float hOverE,
+                     float ecalIso,
+                     float hcalIso,
+                     float trkIso,
+                     float r9,
+                     float sMin,
+                     float sMaj,
+                     unsigned int seedId,
+                     std::vector<float> energyMatrix,
+                     std::vector<float> timingMatrix)
+      : pt_(pt),
+        eta_(eta),
+        phi_(phi),
+        m_(m),
+        sigmaIetaIeta_(sigmaIetaIeta),
+        hOverE_(hOverE),
+        ecalIso_(ecalIso),
+        hcalIso_(hcalIso),
+        trkIso_(trkIso),
+        r9_(r9),
+        sMin_(sMin),
+        sMaj_(sMaj),
+        seedId_(seedId),
+        energyMatrix_(std::move(energyMatrix)),
+        timingMatrix_(std::move(timingMatrix)) {}
+  //default constructor
+  Run3ScoutingPhoton()
+      : pt_(0),
+        eta_(0),
+        phi_(0),
+        m_(0),
+        sigmaIetaIeta_(0),
+        hOverE_(0),
+        ecalIso_(0),
+        hcalIso_(0),
+        trkIso_(0),
+        r9_(0),
+        sMin_(0),
+        sMaj_(0),
+        seedId_(0),
+        energyMatrix_(0),
+        timingMatrix_(0) {}
```

Comment:
Implicit call to default constructors is sufficient
```suggestion
        seedId_(0) {}
```